### PR TITLE
Docker documentation

### PR DESCRIPTION
**Pull request by VenkyOS**

### CODE REVIEW

**Review by mukul-seagate11 — APPROVED** [review]
looks good as tested

File: doc/Release_Build_Creation.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+   
+    drwxr-xr-x 3 root root 4096 Dec 16 05:23 cortx_iso
+   
+    drwxr-xr-x 2 root root 4096 Dec 16 05:49 iso
+    
+#. To list individual component targets, execute the below mentioned command.
+ 
+   ::
+    
+    docker run ghcr.io/seagate/cortx-build:centos-7.8.2003 make help
```
Comment (shailesh-vaidya) [inline doc/Release_Build_Creation.rst:65]: Should we add output of `make help`

```
[root@ssc-vm-1613 cortx-re]# time docker run ghcr.io/seagate/cortx-build:centos-7.8.2003 make help
usage: make "target"

Please clone required component repositories in cortx-workspace directory before executing respective targets.

targets:
    help: print this help message.
    clean: remove existing /var/artifacts/0 directory.
    build: generate complete CORTX build including third-party-deps at "/var/artifacts/0"
    control-path: generate control-path packages. cortx-provisioner, cortx-monitor, cortx-manager, cortx-management-portal and cortx-ha.
    io-path: generate io-path packages. cortx-motr, cortx-s3server and cortx-hare.
    cortx-motr: generate cortx-motr packages.
    cortx-s3server: generate cortx-s3server packages.
    cortx-hare: generate cortx-hare packages.
    cortx-ha: generate cortx-ha packages.
    cortx-management-portal: generate cortx-management-portal packages.
    cortx-manager: generate cortx-manager packages.
    cortx-monitor: generate cortx-monitor packages.
    cortx-posix: generate cortx-posix (NFS) packages.
    cortx-prvsnr: generate cortx-prvsnr packages.
    iso_generation: generate ISO file from release build.
```

**Review by johnbent — COMMENTED** [review]
@shailesh-vaidya , is this a newly added file?  If so, can you please also add a link to it from an appropriate location?  If it is already linked from elsewhere, then this looks good to me!

**Comment by shailesh-vaidya** [comment]
> @shailesh-vaidya , is this a newly added file? If so, can you please also add a link to it from an appropriate location? If it is already linked from elsewhere, then this looks good to me!

Hi @johnbent This is newly added file for steps to use cortx-build docker image. @VenkyOS  Can you please add link to it from an appropriate location.

**Comment by VenkyOS** [comment]
> > @shailesh-vaidya , is this a newly added file? If so, can you please also add a link to it from an appropriate location? If it is already linked from elsewhere, then this looks good to me!
> 
> Hi @johnbent This is newly added file for steps to use cortx-build docker image. @VenkyOS Can you please add link to it from an appropriate location.

@shailesh-vaidya - I will add the relevant link.

File: doc/Release_Build_Creation.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+#. Setup a CentOS 7.8.2003 system.
+
+   - You can use a Virtual Machine (VM) also.
+   
+#. Install the docker packages in the system or VM. Refer to `Docker Installation <https://docs.docker.com/engine/install/centos/>`_.
+
+#. Login to GitHub Docker.
+
+#. Clone the CORTX repository by running the following command.
```
[the remark this answers is not in the document]
Comment (VenkyOS) [reply doc/Release_Build_Creation.rst]: Icorporated.

File: doc/Release_Build_Creation.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+   ::
+   
+    mkdir -p /var/artifacts
+
+#. Build CORTX artifacts using the below mentioned docker.
+
+   ::
+   
+    time docker run --rm -v /var/artifacts:/var/artifacts -v /root/cortx:/cortx-workspace -e GITHUB_TOKEN=$GITHUB_TOKEN ghcr.io/seagate/cortx-build:centos-7.8.2003 make clean build
```
[the remark this answers is not in the document]
Comment (VenkyOS) [reply doc/Release_Build_Creation.rst]: Incorporated.

**Comment by VenkyOS** [comment]
> @shailesh-vaidya , is this a newly added file? If so, can you please also add a link to it from an appropriate location? If it is already linked from elsewhere, then this looks good to me!

@johnbent - I have added a link to this document from the below mentioned page.

[Build_Environment](https://github.com/Seagate/cortx/blob/Docker_Documentation/doc/BUILD_ENVIRONMENT.md)

File: doc/Release_Build_Creation.rst (hunk 1 above)
Comment (VenkyOS) [reply doc/Release_Build_Creation.rst:63]: Incorporated.

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**